specs/sharding/beacon-chain.md

```diff
@@ -103,460 +101,342 @@ The following values are (non-configurable) constants used throughout the specif
 | - | - | - |
 | `PRIMITIVE_ROOT_OF_UNITY` | `7` | Primitive root of unity of the BLS12_381 (inner) modulus |
 | `DATA_AVAILABILITY_INVERSE_CODING_RATE` | `2**1` (= 2) | Factor by which samples are extended for data availability encoding |
-| `POINTS_PER_SAMPLE` | `uint64(2**3)` (= 8) | 31 * 8 = 248 bytes |
+| `POINTS_PER_SAMPLE` | `uint64(2**4)` (= 16) | 31 * 16 = 496 bytes |
 | `MODULUS` | `0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001` (curve order of BLS12_381) |
 
-### Domain types
-
-| Name | Value |
-| - | - |
-| `DOMAIN_SHARD_BLOB` | `DomainType('0x80000000')` |
+## Preset
 
-### Shard Work Status
+### Misc
 
 | Name | Value | Notes |
 | - | - | - |
-| `SHARD_WORK_UNCONFIRMED` | `0` | Unconfirmed, nullified after confirmation time elapses |
-| `SHARD_WORK_CONFIRMED` | `1` | Confirmed, reduced to just the commitment |
-| `SHARD_WORK_PENDING` | `2` | Pending, a list of competing headers |
-
-### Misc
-
-TODO: `PARTICIPATION_FLAG_WEIGHTS` backwards-compatibility is difficult, depends on usage.
-
-| Name | Value |
-| - | - |
-| `PARTICIPATION_FLAG_WEIGHTS` | `[TIMELY_SOURCE_WEIGHT, TIMELY_TARGET_WEIGHT, TIMELY_HEAD_WEIGHT, TIMELY_SHARD_WEIGHT]` |
-
-### Participation flag indices
+| `MAX_SHARDS` | `uint64(2**12)` (= 4,096) | Theoretical max shard count (used to determine data structure sizes) |
+| `ACTIVE_SHARDS` | `uint64(2**8)` (= 256) | Initial shard count |
+| `TARGET_SHARDS` | `uint64(ACTIVE_SHARDS // 2)` (= 256) | Initial shard count |
```

Comment:
```suggestion
| `TARGET_SHARDS` | `uint64(ACTIVE_SHARDS // 2)` (= 128) | Initial shard count |
```

Reply:
Actually this number is currently irrelevant as no EIP1559 here yet. Removing